.github/ISSUE_TEMPLATE/Bug.md

```diff
@@ -5,26 +5,49 @@ about: Report a problem with rclone
 
 <!--
 
-Welcome :-) We understand you are having a problem with rclone; we want to help you with that!
+We understand you are having a problem with rclone; we want to help you with that!
 
-If you've just got a question or aren't sure if you've found a bug then please use the rclone forum:
+**STOP and READ**
+**YOUR POST WILL BE REMOVED IF IT IS LOW QUALITY**:
+Please show the effort you've put in to solving the problem and please be specific.
+People are volunteering their time to help! Low effort posts are not likely to get good answers!
+
+If you think you might have found a bug, try to replicate it with the latest release or latest beta.
+Update your rclone:
+
+    rclone self-update
+or
+    rclone self-update --beta
+
+If your rclone reports `unknown command "self-update"` then you'll need to update manually
+following the install instructions at https://rclone.org/install
+    
+If you can still replicate it or just got a question then please use the rclone forum:
 
     https://forum.rclone.org/
 
-instead of filing an issue for a quick response.
+for a quick response instead of filing an issue on this repo.
 
-If you think you might have found a bug, please can you try to replicate it with the latest beta?
+If nothing else helps, then please fill in the info below which helps us help you.
+**DO NOT REDACT** any information except passwords/keys/personal info.
```

Comment:
Should probably be on a separate line to emphasize it.

Reply:
ok